rust/sbd-client/src/raw_client.rs

```diff
@@ -241,8 +241,7 @@ fn priv_system_tls(
         any(target_os = "windows", target_os = "linux", target_os = "macos",),
     ))]
     roots.add_parsable_certificates(
-        rustls_native_certs::load_native_certs()
-            .expect("failed to load system tls certs"),
+        rustls_native_certs::load_native_certs().certs,
```

Comment:
The `.expect()` here would panic if there had been [an error for *any* certificate](https://github.com/rustls/rustls-native-certs/blob/98fee3f68f41456ec8bb363dce47e3c53a3ea277/src/lib.rs#L139-L144). Such errors [can occur if a certificate has a corrupted trust settings record](https://github.com/rustls/rustls-native-certs/blob/98fee3f68f41456ec8bb363dce47e3c53a3ea277/src/macos.rs#L50-L53).